_api-reference/tasks.md

```diff
@@ -203,6 +219,22 @@ curl -XGET "localhost:9200/_tasks?actions=*search&detailed
 
 ```
 
+### The `resource_stats` object
+
+The `resource_stats` object is only updated for tasks that support resource tracking. These stats are computed based on scheduled thread executions, which includes both threads that have finished working on the task and threads currently working on the task. Because the same thread may be scheduled to work on the same task multiple times, each time a given thread is scheduled to work on a given task is considered a single thread execution.
+
+The following table lists all response fields in the `resource_stats` object. 
+
+Response field | Description |
+:--- | :--- |
+`average` | The average resource usage across all scheduled thread executions. |
+`total` | The sum of resource usages across all scheduled thread executions. |
+`min` | The minimum resource usage across all scheduled thread executions. |
+`max` | The maximum resource usage across all scheduled thread executions. |
+`thread_info` | Thread count-related stats.|
```

Comment:
If this means "count-related stats for the thread", then the hyphenation is correct, but confirm that it shouldn't be "thread-count-related stats".

Reply:
It is "thread-count-related". I'll change.